etc/nebula-graphd.conf.default

```diff
@@ -24,8 +24,8 @@
 ########## networking ##########
 # Meta Server Address
 --meta_server_addrs=127.0.0.1:45500
-# Local ip
---local_ip=127.0.0.1
```

Comment:
Why rename this option?

Reply:
For graphd, `local_ip  `is not used.
Here graphd uses the local ip specified by graph_server_ip to start.